examples.md

```diff
@@ -657,3 +658,35 @@ steps:
       restore-keys: |
         ${{ runner.os }}-mint-
 ```
+
+## * - Bazel
+
+[`bazelisk`](https://github.com/bazelbuild/bazelisk) does not have be to separately downloaded and installed because it's already included in GitHub's `ubuntu-latest` base image.
```

Comment:
can we also point out that its already part of macos-latest as well?

Reply:
done